src/getErrors.js

```diff
@@ -0,0 +1,31 @@
+const getErrors = (text, toCase) => {
+  const availableCases = ['SNAKE', 'PASCAL', 'KEBAB', 'CAMEL', 'UPPER'];
+  const errors = [];
+
+  if (!text) {
+    errors.push({
+      message: 'Text to convert is required. '
+        + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!toCase) {
+    errors.push({
+      message: '"toCase" query param is required. '
+        + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!availableCases.includes(toCase) && toCase) {
+    errors.push({
+      message: 'This case is not supported. '
+        + 'Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
```

Comment:
It's good practice to use template literals like this. It doesn't require to escape special characters (text inside will be as it is including spacing). Also, you can include variables inside. You have the variable cases, and use values from it to have always up-to-date errors

```suggestion
      message: `This case is not supported. Available cases: ${availableCases.join(', ')}.`,
```